pages/instructorAssessment/instructorAssessment.sql

```diff
@@ -104,6 +104,65 @@ WHERE
 ORDER BY
     aar.number;
 
+-- BLOCK assessment_score_histogram_by_date
+WITH assessment_instances_by_user_and_date AS (
+    SELECT
+        ai.user_id,
+        avg(ai.score_perc) AS score_perc,
+        date_trunc('day', date) AS date
+    FROM
+        assessment_instances AS ai
+    WHERE
+        ai.assessment_id = $assessment_id
+    GROUP BY
+        1, 3
+    ORDER BY 3
+),
+dates AS (
+    SELECT DISTINCT ai_by_user_and_date.date
+    FROM
+        assessment_instances_by_user_and_date AS ai_by_user_and_date
+),
+buckets AS (
+    SELECT generate_series(1, 10) AS bucket
+),
+dates_and_buckets AS (
+    SELECT * FROM dates CROSS JOIN buckets
+),
+assessment_instances_by_date_bucketed AS (
+    SELECT
+        count(ai_by_user_and_date.user_id) AS num_users_in_bucket,
+        dates_and_buckets.bucket AS bucket,
+        to_char(dates_and_buckets.date, 'DD Mon YYYY') AS date,
```

Comment:
keep `date` as the actual date variable, use `date_formatted` for the string version